### PR TITLE
Adds the recipe for miniocli.

### PR DESCRIPTION
**Pull request by dipietrantonio**

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you fix style? Normally we'd do it with @spackbot, but when submitting from an organization, you cannot allow us to act on your behalf

**Comment by dipietrantonio** [comment]
Hi @alalazo, sure. I will fix the style on every PR I opened, where needed. I am in the process of pushing all our local additions and improvements back to this repo. I will iterate over all the PRs again in the following days. Thanks for taking the time to review them!

**Comment by dipietrantonio** [comment]
Hi @alalazo, I have fixed almost all style errors. The last one seems to be about the license. To me it looks ok though, like the one on top of other recipes. Can you tell me what's wrong?

While we are at it, what should the license be for Pawsey-created packages?

File: var/spack/repos/builtin/packages/miniocli/package.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/miniocli/package.py]:
```suggestion
# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment (alalazo) [reply var/spack/repos/builtin/packages/miniocli/package.py]: > Can you tell me what's wrong?

See above

> While we are at it, what should the license be for Pawsey-created packages?

License should be the same on every package that is contributed in the builtin repository of Spack

**Review by alalazo — APPROVED** [review]

**Pull request merged**